.github/CODEOWNERS

```diff
@@ -4,74 +4,74 @@
 # First Lets start with areas with no filters or paths
 
 # Area: Performance
-# @lzybkr @SteveL-MSFT @adityapatwardhan
+# @lzybkr @adityapatwardhan
 
 # Area: Portability
-# @BrucePay @JamesWTruher @SteveL-MSFT
+# @BrucePay @JamesWTruher
 
 # Area: Security
-# @TravisEz13 @leeholmes @SteveL-MSFT @PaulHigin @chunqingchen
+# @TravisEz13 @leeholmes @PaulHigin @chunqingchen
```

Comment:
remove leeholmes, he isn't generally available so Paul, Travis, or I will pull him in when needed.  Remove chunqing